sros2/test/sros2/commands/security/verbs/test_generate_policy.py

```diff
@@ -24,6 +24,8 @@
 from std_srvs.srv import Trigger
 
 
+# TODO(ivanpauno): reactivate this test after updating generate policy
+@pytest.mark.skipif(True, reason='temporarily deactivated')
```

Comment:
@ivanpauno FYI `pytest.mark.skip` has the same effect, unconditionally.

Reply:
Done in https://github.com/ros2/sros2/pull/177/commits/070105b15778030a10ff45a98d02964689752822.